Add datalab notebook (#50)
cleanlab/examples: datalab/README.md

> # Training a Swin Transformer model with PyTorch on a subset of Caltech-256 for auditing with Datalab

Reviewer: ```suggestion
# Detecting issues in an image classification dataset (Caltech-256) with Datalab and a Swin Transformer model.
```

> # Detecting issues in an image classification dataset (Caltech-256) with Datalab and a Swin Transformer model.

Reviewer: Add a reference to this example in the table of contents from the main readme in examples/

> This example demonstrates the use of the following module below from cleanlab:

Reviewer: ```suggestion
This example demonstrates cleanlab's Datalab class for auditing an image dataset.
```

> # Detecting issues in an image classification dataset (Caltech-256) with Datalab and a Swin Transformer model.
> This example demonstrates cleanlab's Datalab class for auditing an image dataset.

Reviewer: ```suggestion
# Detecting issues in an image classification dataset (Caltech-256) with Datalab and a Swin Transformer model.

This example demonstrates cleanlab's Datalab class for auditing an image dataset.
```

It is split into two notebooks:

> - [train_image_classifier.ipynb](train_image_classifier.ipynb) - Trains a Swin Transformer model on a subset of Caltech-256

Reviewer: ```suggestion
- [train_image_classifier.ipynb](train_image_classifier.ipynb) - Trains a Swin Transformer classifier model on a subset of Caltech-256
```

  - Install dependencies with:

    ```
    pip install -r requirements-train.txt
    ```

> - [datalab.ipynb](datalab.ipynb) - Runs the dataset through Datalab with the trained model artifacts.
>   - Install dependencies with

Reviewer: ```suggestion
- [datalab.ipynb](datalab.ipynb) - Audits the dataset using Datalab applied to outputs from the trained model.
  - Install dependencies with
```
